src/main/python/aut/__init__.py

```diff
@@ -1,5 +1,39 @@
 from aut.app import Extract_Popular_Images, Write_Gexf, Write_Graphml
 from aut.common import WebArchive
-from aut.udfs import Udf
+from aut.udfs import (
+    compute_image_size,
+    compute_md5,
+    compute_sha1,
+    detect_language,
+    detect_mime_type_tika,
+    extract_boilerplate,
+    extract_date,
+    extract_domain,
+    extract_image_links,
+    extract_links,
+    get_extension_mime,
+    remove_html,
+    remove_http_header,
+    remove_prefix_www,
+)
 
-__all__ = ["Extract_Popular_Images", "Udf", "WebArchive", "Write_Gexf", "Write_Graphml"]
+__all__ = [
+    "Extract_Popular_Images",
```

Comment:
Apologies in advance if this has been hashed out already, but have we agreed on a naming convention for these? Naming seems inconsistent at present...